Resources/config/services.yml

```diff
@@ -107,3 +101,17 @@ services:
 
         tags:
             - {name: 'request.param_converter', converter: 'json_api.request_params.converter.http_request_converter'}
+
+    json_api.datetime_handler:
+        class: Mango\Bundle\JsonApiBundle\Serializer\Handler\DateHandler
+        arguments:
+            - '@jms_serializer.datetime_handler'
+        tags:
+            - { name: jms_serializer.subscribing_handler }
```

Comment:
Do we really need a separate class which does not even adds extra logic (as decorators should), I'd add a CompilerPass which will register subject `DateHandler` with `json:api` tag (review how subscribing_handler are handled to get the clue)

Reply:
@oleg-andreyev it seems that `getSubscribingMethods` is the only way how to declare format and there no other option to declare `DateHandler` for `json:api`